app/views/judgment/judgmentComplete.scala.html

```diff
@@ -0,0 +1,39 @@
+@import java.util.UUID
+@(consignmentReference: String)(implicit messages: Messages, request: RequestHeader)
+    @main("Transfer complete") {
+        @defining(play.core.PlayVersion.current) { version =>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+
+                    <div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-6">
+                        <h1 class="govuk-panel__title">
+                            Transfer complete
+                        </h1>
+                        <div class="govuk-panel__body govuk-!-font-size-27">
+                        Your file have now been transferred to The National Archives.
+                        </div>
+                        <div class="govuk-panel__body govuk-!-font-size-27 govuk-!-margin-top-5">
+                            Consignment reference: <span class="govuk-!-font-weight-bold">@consignmentReference</span>
```

Comment:
According to the mock up, this should be "Reference" rather than, "Consignment reference"